lib/recurly/client.php

```diff
@@ -49,6 +49,7 @@ class Recurly_Client
   const PATH_BILLING_INFO = '/billing_info';
   const PATH_COUPON = '/coupon';
   const PATH_COUPON_REDEMPTION = '/redemption';
+  const PATH_COUPON_REDEMPTIONS = '/redemptions';
```

Comment:
Turns out this path doesn't work. I wasn't testing it properly when I merged it. We'll get it sorted in some upcoming work.